lib/src/utils.dart

```diff
@@ -20,9 +20,11 @@ final Future<bool> supportsIPv6 = () async {
 final Future<bool> supportsIPv4 = () async {
   try {
     var socket = await ServerSocket.bind(InternetAddress.loopbackIPv4, 0);
-    socket.close();
+    unawaited(socket.close());
     return true;
   } on SocketException catch (_) {
     return false;
   }
 }();
+
+void unawaited(Future<void> f) {}
```

Comment:
worth a `// NOTE` explaining why you're not using the impl in pedantic? (avoid more dependencies, I'm guessing...)

Reply:
It's in the commit message and should show up with a `git blame` easy enough.

Reply:
😕 – bleh. Don't make me use `git blame` when I'm scrolling through source!

Reply:
IMO `git blame` is exactly the tool we should use to answer that type of question. It's a detail that is almost never relevant and does not alleviate the risk of introducing a bug.

Reply:
Added a comment.